### PR TITLE
Adds support for firstDay option

### PR DESCRIPTION
**Pull request by beerlington**
I'm not totally sure how to test this since it's just passing an option along to Pikaday and I didn't want to test Pikaday behavior.

######

The firstDay option lets you set the first day of the calendar.

    0 = Sunday
    1 = Monday
    etc...

### CODE REVIEW

File: addon/components/pikaday-input.js [hunk 1]
```diff
@@ -15,7 +16,7 @@ export default Ember.Component.extend({
       onClose: Ember.run.bind(this, this.onPikadayClose),
       onSelect: Ember.run.bind(this, this.onPikadaySelect),
       onDraw: Ember.run.bind(this, this.onPikadayRedraw),
-      firstDay: 1,
+      firstDay: (typeof firstDay !== 'undefined') ? parseInt(firstDay) : 1,
```
Comment (stravid) [inline addon/components/pikaday-input.js]: Please explicitly add the radix: `parseInt(firstDay, 10)`

**Comment by stravid** [comment]
First of all big thanks for implementing this yourself and updating the README :+1: 

Regarding the tests I agree that we should not test Pikaday behaviour. But currently there is no way to tell if some new code or refactoring interferes with this feature. So this is also not an optimal solution.

To prevent future regressions I would write a test where the first day attribute is set to something different than the default. The assertion than ensures that the popup looks correctly.

I don't look at it as "testing Pikaday" but more as "testing if we pass the attribute to Pikaday". If you know a better approach to accomplish this, I'm all ears :)

**Comment by beerlington** [comment]
@stravid Thanks for the feedback. I added a few specs, added the parseInt radix option and rebased. Let me know if that's what you had in mind.

**Comment by stravid** [comment]
@beerlington perfect, thank you so much!

**Pull request merged**